README.md

```diff
@@ -1,13 +1,13 @@
 # Sport-car
 
 ## Idea Inicial 🚀
-Siempre nos ha pasado de querer hacer una excursión a la montaña, o de querer ir a un rocodromo al cual solo pueder ir en algún tipo de vehículo pero no tienes, ni conoces a nadie que tenga y te pueda llevar, dando tambíen un problema en el cual las instituciones no saben donde fundar un recinto para elaborar la actividad, la idea es hacer una aplicación que conecte gente que va a realizar una excursión o actividad física al exterior y tenga hueco en su vehiculo con personas que en ese momento no dispone de capacidad para poder moverse, y de paso hacerse compañeros de excursiones :D
+Siempre nos ha pasado de querer hacer una excursión a la montaña, o de querer ir a un rocódromo al cual solo pueder ir en algún tipo de vehículo pero no tienes, ni conoces a nadie que tenga y te pueda llevar, dando tambíen un problema en el cual las instituciones no saben donde fundar un recinto para elaborar la actividad, la idea es hacer una aplicación que conecte gente que va a realizar una excursión o actividad física al exterior y tenga hueco en su vehiculo con personas que en ese momento no dispone de capacidad para poder moverse, y de paso hacerse compañeros de excursiones :D
```

Comment:
De veras, no metáis estos pequeños cambios en el PR, hacedlo antes, que sólo genera más cosas para revisar que no tienen nada que ver con el objetivo.

Reply:
Si, perdón, entiendo lo que supone ya que yo también he pasado a revisar últimamente cosas de github y supone un problema esto. No lo volveré a hacer. Gracias por la ayuda a mejorar 😄